iib/workers/tasks/utils.py

```diff
@@ -695,6 +696,8 @@ def wrapper(*args, **kwargs):
         finally:
             if request_log_handler:
                 logger.removeHandler(request_log_handler)
+                if worker_config['iib_aws_s3_bucket_name']:
+                    upload_file_to_s3_bucket(log_file_path, f'{request_id}.log', 'request_logs')
```

Comment:
I think before making this call, we need to flush the log handler, something like `request_log_handler.flush()`. Actually, it might be a good idea to do that always, right after the `removeHandler` call.

Reply:
My comment is addressed and can be resolved. I won't resolve this conversation to make it easier for @lipoja to review the changes to also verify his comment is addressed.

(Apologies for mixing conversations. I thought I had created a new one)

Reply:
Thanks, changes were addressed.